plugins/network-elements/nuage-vsp/src/com/cloud/agent/api/element/ImplementVspAnswer.java

```diff
@@ -17,18 +17,27 @@
 // under the License.
 //
 
-package com.cloud.agent.api.guru;
+package com.cloud.agent.api.element;
 
 import com.cloud.agent.api.Answer;
-import com.cloud.agent.api.Command;
 
-public class ReleaseVmVspAnswer extends Answer {
+public class ImplementVspAnswer extends Answer {
```

Comment:
Why does this class exist?  It adds no state or behavior to the ``Answer`` base class.  Why not simply use ``Answer`` in this circumstance?